Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM freedomkk/tomcat-maven:8
+
+ENV BUILD_DIR /tmp/lodestar/
+
+RUN mkdir $BUILD_DIR && \
+  cd $BUILD_DIR && \
+  git clone https://github.com/amalic/lodestar.git
```

Comment:
can it use https://github.com/EBISPOT/lodestar by default?

Reply:
yes